### DISCUSSION TITLE
Cannot download example directory using npx create-next-app

### DISCUSSION THREAD

**Discussion by hengaini2055** [vercel/next.js#24802] [Help]
OS: ubuntu 2004 server , created by multipass from HOST windows 10
Git connect github.com status: command git clone is success
question: Cannot download example 

`npx create-next-app nextjs-blog --use-npm --example blog-starter
npx: installed 1 in 2.584s
Creating a new Next.js app in /home/ubuntu/nextjs-blog.

Downloading files for example blog-starter. This might take a moment.

? Could not download "blog-starter" because of a connectivity issue between your machine and GitHub.
✔ Could not download "blog-starter" because of a connectivity issue between your machine and GitHub.
Do you want to use the default template instead? … yes
Creating a new Next.js app in /home/ubuntu/nextjs-blog.
`

**Comment by chof64** [comment]
Hello, I am having the same issue, however, on mine, the `--example` points to  `https://github.com/vercel/next-learn/tree/master/basics/learn-starter` as per the tutorial (https://nextjs.org/learn/basics/create-nextjs-app/setup).
![Screenshot from 2022-07-30 15-32-31](https://user-images.githubusercontent.com/46257935/181880324-a8ee52e4-8a35-4cd8-a201-b6d7a5373342.png)

But what I noticed is that all the files from the GitHub repo seem to be downloaded. However, dependencies are not installed.

So with the assumption of all the files are fully downloaded, I just manually run `npm install to install dependencies. And proceeded to run `npm run dev` to start the dev server. And it worked!
![Screenshot from 2022-07-30 15-41-59](https://user-images.githubusercontent.com/46257935/181880396-14d1d958-4340-434b-85a1-a93021fcf27a.png)

It seems that there's some bug in the script that even though it already downloaded the repo, returns an error even though the download is complete. And with this error, the rest of the script (dependency installation) is not run.

**Reply by arturo-guerra** [reply]
Thanks! I had the same issue and running the installation manually worked!
